bindings/python/src/session.cpp

```diff
@@ -667,7 +695,13 @@ void bind_session()
         .def("is_dht_running", allow_threads(&lt::session::is_dht_running))
         .def("set_dht_settings", allow_threads(&lt::session::set_dht_settings))
         .def("get_dht_settings", allow_threads(&lt::session::get_dht_settings))
-#endif
+        .def("dht_get_immutable_item", &dht_get_immutable_item)
+        .def("dht_get_mutable_item", &dht_get_mutable_item)
+        .def("dht_put_immutable_item", &dht_put_immutable_item)
+        .def("dht_put_mutable_item", &dht_put_mutable_item)
```

Comment:
instead of writing wrappers for these, you can cast the function to the overload you're interested in. See example on line 533.